core/iwasm/common/wasm_runtime_common.h

```diff
@@ -124,6 +138,17 @@ STORE_U8(void *addr, uint8_t value)
     } while (0)
 #endif
 
+static inline V128
+GET_V128_FROM_ADDR(uint32 *addr)
+{
+    V128 ret;
+    ret.i32x4[0] = addr[0];
+    ret.i32x4[1] = addr[1];
+    ret.i32x4[2] = addr[2];
+    ret.i32x4[3] = addr[3];
+    return ret;
+}
```

Comment:
Not very sure why the CIs can run successfully, but I think that V128 may be undeclared in many targets/platforms when WASM_CPU_SUPPORTS_UNALIGNED_ADDR_ACCESS is 0, and compilation error may be reported, so here how about wrapping this function with `#if WASM_ENABLE_SIMD != 0 .. #endif`?

Reply:
`V128` is unconditionally declared in the `wasm.h` file so I don't think there's a need for that extra ifdef.

Reply:
Hmm, got it, I wrongly thought that it is the type `v128` which is defined in `wasm_runtime_common.c` to support `wasm_runtime_invoke_native`, and is defined only when SIMD is enabled.